pkg/controllers/workapplier/controller.go

```diff
@@ -210,47 +229,35 @@ type Reconciler struct {
 	joined                       *atomic.Bool
 	parallelizer                 *parallelizer.Parallerlizer
 
-	availabilityCheckRequeueAfter time.Duration
-	driftCheckRequeueAfter        time.Duration
+	requeueRateLimiter *RequeueFastSlowWithExponentialBackoffRateLimiter
 }
 
+// NewReconciler returns a new Work object reconciler for the work applier.
+//
+// TO-DO (chenyu1): evaluate if KubeFleet needs to expose the requeue rate limiter
```

Comment:
yes, next PR.

Reply:
N/A -> per earlier discussion, the params have been exposed in this PR.